osm2pgsql.lua

```diff
@@ -672,20 +712,57 @@ function osm2pgsql.process_relation(object)
         end
     end
 
---     Adds dispute=yes to any ways part of a boundary=disputed relation
+-- Adds dispute=yes to any ways part of a boundary=disputed relation
     if (type == 'linestring' or type == 'boundary') and object.tags.boundary == 'disputed' then
         for _, member in ipairs(object.members) do
             if member.type == 'w' then
                 if not disputed[member.ref] then
                     disputed[member.ref] = {}
                 end
-                disputed[member.ref] = object.id
+                disputed[member.ref] = object.tags
             end
         end
+        output_hstore.dispute = 'yes'
     end
 
     if type == 'boundary' and (object.tags['ISO3166-1'] == 'MO' or object.tags['ISO3166-1'] == 'HK') then
```

Comment:
Add comment about why we're doing this

Reply:
@jeffdefacto mind adding a comment here?